src/lib/nvme_pt_ioctl.c

```diff
@@ -257,6 +258,12 @@ static void cpy_opalv200_feat(struct sed_opal_level0_discovery *discv,
 		sizeof(struct opalv200_supported_feat));
 }
 
+static void cpy_cnl_feat(struct sed_opal_level0_discovery *discv, void *feat)
+{
+	memcpy(&discv->sed_cnl, (struct cnl_feat *)feat,
```

Comment:
fix missiing whitespace between casting and param name

Reply:
will fix it